.env.example

```diff
@@ -41,4 +41,4 @@ EMAIL_PORT='1025'
 SECRET_KEY_BASE=[??????]
 
 ## For Google Analytics
```

Comment:
```suggestion
## Google Tag Manager ID, required to enable Google Analytics
```

Think it might be helpful to clarify we aren't after an Analytics ID here.